mail_tracking/tests/test_mail_tracking.py

```diff
@@ -169,7 +169,7 @@ def test_email_cc(self):
             'login': 'sender-test',
         })
         message = self.recipient.sudo(user=sender_user).message_post(
-            body='<p>This is a test message</p>',
+            body=_('<p>This is a test message</p>'),
```

Comment:
Why translated? This translation is not even loaded in POT file. If you have being testing in local, you must do it always in English.

Reply:
If not use it travis shows: https://travis-ci.org/github/OCA/social/jobs/664560869#L319

Reply:
That's only a warning, not an error, and in this case, it's not correct. We should fix it in MQT, but any way, please remove it and you can flag the line with `# pylint: disable=C8107` for avoiding the warning message.